modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+|`ssl-ca-directory` |A workspace containing CA certificates. If you provide this workspace, Git uses these certificates to verify the peer when interacting with remote repositories using HTTPS.
+|`output` |A workspace that contains the fetched git repository, data will be placed on the root of the workspace or on the relative path defined by the `SUBDIRECTORY` parameter.
+|===
+
+.Results that the `git-clone` task returns
+[options="header"]
+|===
+| Result | Type | Description
+|`COMMIT` |`string` |The SHA digest of the commit that is at the HEAD of the current branch in the cloned Git repository.
+|`URL` |`string` |The URL of the repository that was cloned.
+|`COMMITTER_DATE` |`string` |The epoch timestamp of the commit that is at the HEAD of the current branch in the cloned Git repository.
+|===
+
+.Changes from the `git-clone` `ClusterTask`
+
+* All parameter names were changed to uppercase.
+* All result names were changed to uppercase.
+* The `gitInitImage` parameter was removed.
+
+[discrete]
+[id="op-taskref-kn_{context}"]
+== kn
+
+The `kn` task uses the `kn` command line utility to complete operations on Knative resources, such as services, revisions, or routes.
+
+.Example usage of the `kn` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+ name: kn-run
+spec:
+ pipelineSpec:
+   tasks:
+   - name: kn-run
+     taskRef:
+       resolver: cluster
+       params:
+       - name: kind
+         value: task
+       - name: name
+         value: kn
+       - name: namespace
+         value: openshift-pipelines
+     params:
+     - name: ARGS
+       value: [version]
+----
+
+.Supported parameters for the `kn` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`ARGS` |The arguments for the `kn` utility. |`array` |`- help`
+|===
+
+.Changes from the `kn` `ClusterTask`
+
+* The `KN_IMAGE` parameter was removed.
+
+[discrete]
+[id="op-taskref-kn-apply_{context}"]
+== kn-apply
+
+The `kn-apply` task deploys a specified image to a Knative Service. This task uses the `kn service apply` command to create or update the specified Knative service.
+
+.Example usage of the `kn-apply` task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+ name: kn-apply-run
+spec:
+ pipelineSpec:
+   tasks:
+   - name: kn-apply-run
+     taskRef:
+       resolver: cluster
+       params:
+       - name: kind
+         value: task
+       - name: name
+         value: kn-apply
+       - name: namespace
+         value: openshift-pipelines
+     params:
+     - name: SERVICE
+       value: "hello"
+     - name: IMAGE
+       value: "gcr.io/knative-samples/helloworld-go:latest"
+----
+
+.Supported parameters for the `kn-apply` task
+[options="header"]
+|===
+| Parameter | Description | Type | Default value
+|`SERVICE` |The Knative service name |`string` |
```

Comment:
These descriptions don't end in periods, but I think most of your other descriptions in this file do. Just double check and try to be consistent if possible.